src/launcher/main.cpp

```diff
@@ -485,12 +523,19 @@ int main(int argc, const char** argv) {
         return 1;
     }
 
-    setup_output_files(pid);
-    setup_lib_path();
+    if (jattach_cmd == kEmpty) {
+        setup_output_files(pid);
+    } else if (params != kEmpty) {
+        fprintf(stderr, "Warning: --jattach-cmd was given, these parameters will be ignored: %s\n", params.str());
```

Comment:
Like suggested [here](https://github.com/Granulate/async-profiler/pull/7/files#r1189209500) let's be more strict and `exit(1)` here

Reply:
Right.
Actually used `return 1` to be consistent with rest of code in this routine.